version/version_base.go

```diff
@@ -8,6 +8,7 @@ var (
 	// Whether cgo is enabled or not; set at build time
 	CgoEnabled bool
 
+
```

Comment:
This line should be removed and is causing a CI failure due to a `gofumpt` diff. You can just run `make fmt` to fix it.